maas/models.py

```diff
@@ -5,6 +5,14 @@
 
 class MaasOperator(models.Model):
     name = models.CharField(verbose_name=_("name"), max_length=64)
+    identifier = models.CharField(
+        verbose_name=_("identifier"),
+        max_length=64,
+        help_text=_(
+            "Value is sent in an API call to a ticketing system. Identifies the MaaS "
+            "operator in a ticketing system."
+        ),
```

Comment:
We'll probably want `unique=True` here(?)

Reply:
Maybe, I'll make it unique.